src/resources/level.py

```diff
@@ -26,19 +26,19 @@ def generate_level(self, x: int, y: int) -> None:
 
     def add_doors(self, doors: list) -> None:
```

Comment:
This methods expects doors as a list but is actually called with len(doors) in the constructor. You should change the type hint to int.

Reply:
Fixed